Components/Queues/Drivers/PhpAmqpLibDriver.php

```diff
@@ -98,30 +75,32 @@ class PhpAmqpLibDriver extends Service implements PurgeableQueueDriverInterface
      * PhpAmqpLibDriver constructor.
      * @param QueueManager $manager
      */
-    public function __construct(PhpAmqpLibQueueManager $manager)
+    public function __construct(PhpAmqpLibQueueManager $manager, string $format = null)
```

Comment:
I see you add the format in the constructor, but I don't see the place where from the configuration this information is passed to the constructor. Based on that, it is always using json, but the format specified by the user in the driver config could be different than json

Reply:
Ok, it was changed.